Formula/czmq.rb

```diff
@@ -22,28 +21,26 @@ class Czmq < Formula
   end
 
   option :universal
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
Same situation as zeromq. It enables additional APIs. As an example,
https://github.com/zeromq/czmq/blob/58d0093d971d4563db631980ba7ff76c916d6a0a/src/zgossip_msg.h